modules/agentless-s3-bucket/main.tf

```diff
@@ -0,0 +1,73 @@
+locals {
+  dd_tags = {
+    Datadog                 = "true"
+    DatadogAgentlessScanner = "true"
+  }
+}
+
+data "aws_region" "current" {}
+
+resource "aws_s3_bucket" "bucket" {
+  bucket_prefix = "datadog-agentless-scanning-"
+  tags          = merge(var.tags, local.dd_tags)
+}
+
+resource "aws_s3_bucket_public_access_block" "bucket_access_block" {
+  bucket = aws_s3_bucket.bucket.id
+
+  block_public_acls       = true
+  block_public_policy     = true
+  ignore_public_acls      = true
+  restrict_public_buckets = true
+}
+
+resource "aws_s3_bucket_lifecycle_configuration" "bucket_lifecycle" {
+  bucket = aws_s3_bucket.bucket.id
+  rule {
+    status = "Enabled"
+    id     = "expire_all_files"
+    expiration {
+      days = 2
```

Comment:
any specific reason why we have 2 days? (if yes perhaps a comment would be helpful, it's also probably worth extracting it as a variable input to the module)

Reply:
Added the comment at the top of the resource.